Neuron/Sections/Transaction/Details/TransactionDetailsViewController.swift

```diff
@@ -83,27 +83,28 @@ class TransactionDetailsViewController: UITableViewController {
 
         switch transaction.status {
         case .success:
-            statusLabel.text = "TransactionStatus.success".localized()
+            statusLabel.text = transaction.to.count > 0 ? "TransactionStatus.success".localized() : "Transaction.Details.contractCreationSuccess".localized()
```

Comment:
`transaction.to.count > 0` is used multiple times. Should extract it as a function or property (perhaps called `isNormalTx` or `isContractCreation`).